src/components/Autocomplete/Autocomplete.tsx

```diff
@@ -0,0 +1,106 @@
+import { useState, useCallback, useMemo } from 'react';
+import classNames from 'classnames';
+import debounce from 'lodash.debounce';
+import { Person } from '../../types/Person';
+import './Autocomplete.scss';
+
+const SEX_MALE = 'm';
+const SEX_FEMALE = 'f';
+const MS_IN_SEC = 1000;
+const DEFAULT_DELAY = 1;
```

Comment:
One constant is enough for the delay
```suggestion
const SEARCH_DELAY = 1000;
```